src/mobile/android/gradlew.bat

```diff
@@ -46,7 +46,7 @@ echo location of your Java installation.
 goto fail
 
 :init
-@rem Get command-line arguments, handling Windows variants
+@rem Get command-line arguments, handling Windowz variants

```

Comment:
I think https://github.com/iotaledger/trinity-wallet/pull/438/commits/7f4aa0fbd05c36cbf4a5f7d2af5c0623ff9ae133 didn't make it to this branch 😄 
```suggestion
@rem Get command-line arguments, handling Windows variants
```